### ISSUE TITLE
Wish: A menu item/hotkey to toggle the host cursor

### ISSUE THREAD

**Issue by xandark** [audetto/AppleWin#161]
I'm using https://github.com/a2stuff/a2d , it is a desktop GUI. I'd like to be able to hide my Linux cursor so that I can see the Apple II desktop's cursor. It'd be great if I could press a hotkey that I could press that would toggle the visibility of the Linux cursor, which is obscuring the a2d cursor. 

![Screenshot 2024-04-21 (11:28:35)](https://github.com/audetto/AppleWin/assets/5048913/60511c89-26e7-47e5-91a1-3f5ff6b997a7)

**Comment by xandark** [comment]
Maybe I wasn't clear, let me try again. I want to be able to hide my Linux pointer so that I can see the pointer in the Apple II desktop. In the screenshot, you can see my Linux pointer right on top of the Apple II desktop's pointer. I want to hide the Linux one.

**Comment by audetto** [comment]
We are talking about the "arrow", the mouse pointer?

If so, this is exactly what I have done.
Not a menu item, because otherwise, I need more logic to show it when it is used to interact with the menu.
You will have to use 'Ctrl-F9' to toggle.

**Comment by xandark** [comment]
Yes, the arrow, the mouse pointer, sorry for not being more clear.  

Okay, I didn't know that Ctrl-F9 toggles the mouse pointer, it's not in the Shortcuts window on the row for F9. But even that aside, pressing Ctrl-F9 doesn't hide my mouse pointer. This is my system:

Operating System: KDE neon 6.0
KDE Plasma Version: 6.0.4
KDE Frameworks Version: 6.1.0
Qt Version: 6.7.0
Kernel Version: 6.5.0-28-generic (64-bit)
Graphics Platform: Wayland

Any ideas?

**Comment by xandark** [comment]
IGNORE THAT LAST MESSAGE. I see that I need to do a pull, duh.

**Comment by xandark** [comment]
Works great!
